data/test/units.cfg

```diff
@@ -5,12 +5,32 @@
 	
 		[race]
 		id=test
-		name= "race^Test"
-		female_name= "race+female^Test"
-		plural_name= "race^Tests"
-		description= "This is a test unit race"
+		name= _ "Language"
+		female_name= _ "Language"
+		plural_name= _ "Language"
+		description= _ "Language"
```

Comment:
It's fine to use the original strings but just make them translatable. Since this file declares its textdomain to be `wesnoth-test`, nothing in here will be added to the gettext files, which don't scrape that textdomain. (This also means there's no need to use the `race^` prefix.)

Reply:
Alright I deleted the prefixes so it should be fine now